upt_macports/templates/base.Portfile

```diff
@@ -4,16 +4,22 @@ PortSystem          1.0
 {% block portgroup %}
 {% endblock %}
 
+# Portfile generated/maintained using the Universal Packaging Tool (upt)
 {% block nameversion %}
 {% endblock %}
 
-maintainers         nomaintainer
-description         {{ pkg.upt_pkg.summary }}
-long_description    ${description}
+# Please adapt the lines with a "TODO" statement and check the guide for more information:
+# https://guide.macports.org/#reference.keywords
 
+# categories-append # TODO
 platforms           darwin
-# uncomment and edit the following line depending on whether the port installs architecture-dependent files
+# TODO: uncomment the following line if no architecture-dependent files are installed, otherwise remove
 # supported_archs     noarch
+license             {{ pkg.licenses }}
+# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer # TODO
```

Comment:
I would also keep a line for `nomaintainer` and let the user decide between the two.
It doesn't serve anyone to force users to assume maintainership when they are not comfortable to do so and not willing to take care of the port later.

Reply:
yes, that's a valid point - how about something like this?
```
# TODO: either add your e-mail address and GitHub handle or set to nomaintainer
# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer
# maintainers         nomaintainer
```

Reply:
One further idea. At the end the port would ideally not contain any comments. So maybe the `TODO`s are a bit superfluous. Maybe something along those lines?

```
## If you would like to keep maintaining this port in the future as described in
##     https://guide.macports.org/chunked/project.contributing.html
## please add your contacts (e-mail, github handle), else keep `nomaintainer`.
# maintainers         {example.org:julesverne @github-handle} openmaintainer
# maintainers         nomaintainer
```

And then also ask users in a comment somewhere on top to go through the commented-out sections and check them out (and ideally remove most comments).